src/Plugin/Action/RemoveOgMembershipRole.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Drupal\og\Plugin\Action;
+
+use Drupal\og\Entity\OgMembership;
+use Drupal\og\Entity\OgRole;
+
+/**
+ * Removes a role from a user.
+ *
+ * @Action(
+ *   id = "og_membership_remove_role_action",
+ *   label = @Translation("Remove a role from the selected members"),
+ *   type = "og_membership"
+ * )
+ */
+class RemoveOgMembershipRole extends ChangeOgMembershipRoleBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function execute(OgMembership $member = NULL) {
+    if (!$member) {
+      return;
+    }
+    $rid = $this->configuration['rid'];
+    $roles = $member->getRoles();
```

Comment:
Good suggestion but this highlights a DX issue, why do we need to load the role in order to remove it? I solved it by introducing a new `revokeRoleById()` method, so this can be reduced to a single line of code:

```
$membership->revokeRoleById($rid)->save()
```